calico-enterprise/reference/resources/globalnetworkpolicy.mdx

```diff
@@ -187,3 +187,23 @@ Application layer policy match criteria are supported with the following restric
 | Datastore type           | Create/Delete | Update | Get/List | Notes |
 | ------------------------ | ------------- | ------ | -------- | ----- |
 | Kubernetes API datastore | Yes           | Yes    | Yes      |
+
+#### List filtering on tiers
+List and Watch operations may specify label selector or field selector to filter GlobalNetworkPolicies on tiers returned by the API server.
+When no selector is specified the API server returns all GlobalNetworkPolicies from all tiers that the user has access to.
+
+##### Field selector
+When using the field selector, supported operators are = and ==
+
+Example to retrieve all GlobalNetworkPolicies in default tier:
+```
+kubectl get globalnetworkpolicy --field-selector spec.tier=default
+```
+
+##### Label selector
+When using the label selector, supported operators are =, == and IN
+
+Example to retrieve all GlobalNetworkPolicies in default and net-sec tier:
+```
+kubectl get globalnetworkpolicy -l 'projectcalico.org/tier in (default, net-sec)'
+```
```

Comment:
```suggestion
##### Label selector

When using the label selector, supported operators are `=`, `==` and `IN`.

The following example shows how to retrieve all `GlobalNetworkPolicy` resources in the `default` and `net-sec` tiers:

```bash
kubectl get globalnetworkpolicy -l 'projectcalico.org/tier in (default, net-sec)'
```
```